db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -28,7 +28,7 @@ def add_columns
 
   def update_columns
     say_with_time("Inline ems_id in containers") do
-      Container.includes(:container_definitions => :container_group).all.each do |container|
+      Container.includes(:container_definition => :container_group).all.each do |container|
```

Comment:
The only other thing I can think of is that ar_virtual is in the way in the ManageIQ application because it patches the preloader.  @kbrock Maybe you have some insight here?